tests/stack/test_api.py

```diff
@@ -154,7 +158,11 @@ def get_job(self, job_id, headers=None):
             return self.jobs_in_databricks[job_id]
 
     def reset_job(self, data, headers=None):
-        if data[api.JOBS_RESOURCE_JOB_ID] not in self.jobs_in_databricks:
+        if data[api.JOBS_RESOURCE_JOB_ID] == self.deleted_job_id:
+            raise HTTPError('Job ID could not be found in the workspace while '
```

Comment:
why do you need this change here? The test job client should not be affected by changes in stack CLI right?